definitions.asmdef

```diff
@@ -1,13 +1,22 @@
 {
     "name": "nl.elraccoone.tweens",
-    "references": [],
-    "optionalUnityReferences": [],
+    "rootNamespace": "",
+    "references": [
+        "GUID:df380645f10b7bc4b97d4f5eb6303d95"
+    ],
     "includePlatforms": [],
     "excludePlatforms": [],
     "allowUnsafeCode": false,
     "overrideReferences": false,
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": []
-}
+    "versionDefines": [
```

Comment:
What exactly does this do? I never used references to other dependencies. Does this mean `com.unity.render-pipelines.core` becomes a required dependency of this module?

Reply:
Ah -- I think I understand. When this module is available, the SRP_AVAILABLE assembly definition will be set, right?

Reply:
Yes, the preprocessor symbol will be generated only if the SRP Core package is available. If not, then the tween class will not be available.

Sorry, I forgot to update the tests. The SRP_AVAILABLE condition also needs to be checked against the test case:

`#if SRP_AVAILABLE
      if (GUILayout.Button ("VolumeWeight"))
        this.target.TweenVolumeWeight (1, 1).SetFrom (0);
#endif`

Reply:
Thanks for clearing that up! I'll release you contribution. 🚀